### DISCUSSION TITLE
How to ensure that images do not break numbering?

### DISCUSSION THREAD

**Discussion by hbcartojano** [alshedivat/al-folio#2389] [Q&A] [answered]
Hello,

I'm creating a post from the `2023-03-20-table-of-contents.md` sample post. It seems like inserting any sort of image breaks numbering?

For example, in the post content

`1. Create a Cloudfront distribution. 

{% include figure.liquid loading="eager" path="assets/img/01/aws_cloudfront_2.png" class="img-fluid rounded z-depth-1" %}

1. Create an AWS IAM user. `

It would render both numbers starting from 1. This holds the same if I add headings of any sort. Is there a way to insert images that would not break numbering?

**Comment by george-gca** [comment, marked as answer]
The possible solutions I found were:
- use `{:start="2"}` ([reference](https://stackoverflow.com/a/41575690/14459430))
- use `1\. Some heading` to avoid rendering it as a list ([reference](https://stackoverflow.com/a/63226685/14459430))

**Reply by hbcartojano** [reply]
The first one works for me. Thanks!

Just as a follow-up, do you know any way to continue unordered lists without losing their level or indentation after inserting an image too?

**Reply by george-gca** [reply]
For unordered list you can simply start a new one. Now if this level is nested, I don't know how to do that. Maybe someone on stackoverflow knows the answer, but in my quick search I couldn't find anything.
